apps/api/src/app/environments/environment.ts

```diff
@@ -119,7 +119,7 @@ const prodConfig = () => ({
   },
   firearmLicense: {
     firearmLicensePassTemplateId: process.env.FIREARM_LICENSE_PASS_TEMPLATE_ID,
-    pkPassApiKey: process.env.VE_PKPASS_API_KEY,
+    pkPassApiKey: process.env.RLS_PKPASS_API_KEY,
```

Comment:
Fyi, we should move most configs like these from environment files and `ClientModule.register` functions to use `@island.is/nest/config`.

Reply:
Yeah, I agree, there just was an issue with config injections into a manually created client